src/api/api.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const appId = 'aBI9ITaYCitPj6DreagZ';
+
+const getBooks = async () => axios.get(`${url}/apps/${appId}/books`).then((result) => {
+  const books = [];
+  if (result.status === 200) {
+    const { data } = result;
+    // eslint-disable-next-line no-restricted-syntax
+    for (const [key, value] of Object.entries(data)) {
+      const id = key;
+      const obj = value;
+      const book = {
+        item_id: id,
+        title: obj[0].title,
+        category: obj[0].category,
+      };
+      books.push(book);
+    }
+  }
+  return books;
+});
+
+const addBook = async (book) => axios
+  .post(`${url}/apps/${appId}/books`, {
+    item_id: book.item_id,
+    title: book.title,
+    category: book.category,
+  })
+  .then((result) => result.data);
+
+const removeBook = async (id) => axios
+  .delete(`${url}/apps/${appId}/books/${id}`, {
+    item_id: id,
+  })
+  .then((result) => result.data)
+  .catch(() => 'error');
```

Comment:
**Changes Required**
- [ ] The `async` keyword is redundant as you're using the `promise.then` syntax. You can choose to use the `async await` syntax or stick to the `promise.then` without the async keyword. Let us try to make changes where in all occurrence of this. This will allow us follow good JavaScript practices.